foreman/data_refinery_foreman/foreman/management/commands/feed_the_beast.py

```diff
@@ -30,7 +30,7 @@ def handle(self, *args, **options):
 
         BATCH_SIZE = 1000
         batch_index = 0
-        batch_accessions = all_accessions[batch_index * BATCH_SIZE:BATCH_SIZE]
+        batch_accessions = all_accessions[0:BATCH_SIZE]
```

Comment:
This change won't actually do anything. You could start at batch_index if you want to avoid the 0. 🤷‍♂

Reply:
I don't, this was just for clarity!